_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
+
+- [Endorsement](#Endorsement)
+- [Outreach](#outreach)
+- [Feedback](#feedback)
+
+### Endorsement
+
+To be official, strategies, actions and other initiatives, must be endorsed or approved by management (DG, CIO, DM, ..).
+The following are way to get things endorsed.
+
+#### CIOEC Endorsement
```

Comment:
Should we define what we want CIOEC Endorsement for? Is it to launch a new project? Dedicate our time to a given initiative? Only when we're seeking funding? Etc ...

Reply:
This is the communication strategy for all our strategies and initiatives.

Reply:
Agree with @smellems , specific strategy/projects will warrant different endorsement levels, and so will target different audience, and so will need different communcation methods to use.